ci/travis_jobs/docker_run_metplus.sh

```diff
@@ -5,8 +5,11 @@
 
 VOLUMES=$3
 
+echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USERNAME" --password-stdin
```

Comment:
You shouldn't need to authenticate to pull the repository since it is public. You only need to do that for pushing. Please remove.

Reply:
Ok. I saw somewhere that I needed to do it for pushes and pulls, but I'll remove it